.github/workflows/python_unit_test.yaml

```diff
@@ -7,14 +7,33 @@ on:
         description: 'A URL identifying a Bash script that can run commands needed for service-specific test setup'
         required: true
         type: string
+      python-versions:
+        description: 'The version of Python to run the unit tests against. Expected to be a string representing an array of versions, this construct is needed due to limitations with Github Actions'
+        required: false
+        type: string
+        default: "['3.9']"
     
 env: 
   PGPASSWORD: postgres
 
 jobs:
+  process_versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - id: set-matrix
+      run: echo "::set-output name=matrix::${{inputs.python-versions }}"
+
+
   run_tests:
+    needs: process_versions
     runs-on: ubuntu-latest
     
```

Comment:
this line is hardcoded with "fence": https://github.com/uc-cdis/.github/blob/826ccf7/.github/workflows/python_unit_test.yaml#L68